assets/scss/00-base/mixins/_ma-typography.scss

```diff
@@ -2,41 +2,33 @@
   font-size: 3rem;
   line-height: 1.07;
 
-  @media ($bp-medium-min){
+  @media ($bp-medium-min) {
     font-size: 3.25rem;
   }
 
-  @media ($bp-large-min){
-    font-size: 4rem;
-  }
-
-  @media ($bp-x-large-min){
-    font-size: 4.375rem;
+  @media ($bp-x-large-min) {
+    font-size: 3.5rem;
   }
 }
 
 @mixin ma-h2 {
-  font-size: 2.25rem;
-  line-height: 1.222222;
-
-  @media ($bp-small-min){
-    font-size: 2.688rem;
-  }
+  font-size: 2rem;
+  line-height: 1.2;
+  margin-bottom: 1rem;
 }
 
 @mixin ma-h3 {
-  font-size: 2rem;
+  font-size: 2.125rem;
```

Comment:
see comment above re: line 15

Reply:
Is there a reason why h3 is bumped up vs down

Reply:
The h2s were changed based on the spreadsheet Colin posted here: https://docs.google.com/spreadsheets/d/1BMQYeIREZv5MoGRMyw2VcJDlEhY9Lotc3MIbYbbVFiw/edit#gid=0 on August 13th. They were then bumped down in size per the review document here: https://docs.google.com/document/d/1IHHD__dIwiS94dFCo-5xgHkd2mrmlchfqVOuuz1pkGQ/edit# on October 25th. 
Honestly, I'm not sure which comment the second change was in response to.